### PR TITLE
Update now value in SyncStatusDisplay

### LINKED ISSUE #11880: Learner’s relative time since last sync doesn’t update as time passes
Spotted by @bjester in 0.16 sync session Feb 14 2024

## Observed behavior

Unless the sync status changes, the relative time since last sync doesn’t change over time as time passes. It seems like there is a bug or inconsistency in our [`<SyncStatusDisplay/>` component's use of relative time](https://github.com/learningequality/kolibri/blob/release-v0.16.x/kolibri/core/assets/src/views/SyncStatusDisplay.vue#L82).

## Errors and logs
no-relative-time

## Expected behavior
While we are already referencing the relative time, we need to fix it if it not working, to update the sync status in between changes in the sync status itself. Another possibility is that we need to change the polling within the side panel, to provide more consistent updates to calculate the relative time, and call the function that calculates it more often. However, we do not want to be polling. If that seems to be the better fix after further investigation, there should be some proposal raised in the issue comments for team discussion before proceeding.

## User-facing consequences
Confusion about when the last sync happened due to slow/inaccurate UI status.

## Steps to reproduce
1. Set up two Kolibris - one as a Learn Only Device (LOD) and one as a full facility. The LOD should be set up to sync back the main facility
2. Assign some resources to the Learner
3. Observe the accuracy of sync status and the calculation of relative time 
4. Repeat if necessary 

Notes:
- opening and closing the side panel does impact the calculation - see the side panel code and sync status indicator code for more
- testing by mocking a slower connection (i.e. Slow 3G) may be helpful in ensuring that the sync happens slowly enough that there is time to observe and troubleshoot

## Context
Kolibri 
release-v0.16.x

### PR DESCRIPTION
**Pull request by AlexVelezLl**
## Summary

Update now value in SyncStatusDisplay component.

https://github.com/learningequality/kolibri/assets/51239030/24946819-30e2-4209-bf46-cb0431036cbf

## References

Closes #11880.

## Reviewer guidance

1. Set up two Kolibris - one as a Learn Only Device (LOD) and one as a full facility. The LOD should be set up to sync back the main facility
2. Assign some resources to the Learner
3. Observe the accuracy of sync status and the calculation of relative time
4. Repeat if necessary

## Testing checklist

- [ ] Contributor has fully tested the PR manually
- [ ] If there are any front-end changes, before/after screenshots are included
- [ ] Critical user journeys are covered by Gherkin stories
- [ ] Critical and brittle code paths are covered by unit tests

## PR process

- [x] PR has the correct target branch and milestone
- [x] PR has 'needs review' or 'work-in-progress' label
- [x] If PR is ready for review, a reviewer has been added. (Don't use 'Assignees')
- [ ] If this is an important user-facing change, PR or related issue has a 'changelog' label
- [ ] If this includes an internal dependency change, a link to the diff is provided

## Reviewer checklist

- Automated test coverage is satisfactory
- PR is fully functional
- PR has been tested for [accessibility regressions](http://kolibri-dev.readthedocs.io/en/develop/manual_testing.html#accessibility-a11y-testing)
- External dependency files were updated if necessary (`yarn` and `pip`)
- Documentation is updated
- Contributor is in AUTHORS.md

### CODE REVIEW

File: kolibri/core/assets/src/composables/useNow.js [hunk 1]
```diff
@@ -0,0 +1,21 @@
+import { now as getNow } from 'kolibri.utils.serverClock';
+
+import { ref, onMounted, onUnmounted } from 'kolibri.lib.vueCompositionApi';
+
+export default function useNow(interval = 10000) {
```
Comment (rtibbles) [inline kolibri/core/assets/src/composables/useNow.js:5]: To justify the composable, we could use this in the ElapsedTime component as well?

Comment (AlexVelezLl) [reply kolibri/core/assets/src/composables/useNow.js:5]: I created a follow-up issue for this in #12114

**Review by rtibbles — COMMENTED** [review]
Code wise, this is good to go, so just want confirmation of the manual fix from QA!

**Comment by pcenov** [comment]
Hi @AlexVelezLl - I confirm that the syncing status is working correctly. Good to go!

**Review by radinamatic — APPROVED** [review]
Manual QA passes, good to go! 💯 👏🏽 :shipit:

**Comment by AlexVelezLl** [comment]
Thanks! @pcenov @radinamatic @rtibbles :tada:

**Pull request merged**